src/convertToObject.js

```diff
@@ -12,7 +12,33 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+
+  const res = styles.map(el => el.split(':'));
+
+  const clear = res.map(element =>
+    element.map(part => part.trim())
+  );
+
+  // const stylesList = {};
+
+  const filtered = clear.filter((item) => (
```

Comment:
let's chain everything instead of creating so many variables
(e.g `styles.map().map().filter()`)